server/routes/analyticsRouter.test.ts

```diff
@@ -0,0 +1,41 @@
+import type { Express } from 'express'
+import request from 'supertest'
+
+import { appWithAllRoutes } from './testutils/appSetup'
+
+let app: Express
+
+beforeEach(() => {
+  app = appWithAllRoutes({})
+  app.locals.featureFlags.showAnalytics = true
```

Comment:
It's nice how this makes it easy to change the flag (and all locals in general) 👍